rust/syntect-plugin/README.md

```diff
@@ -0,0 +1,12 @@
+# Xi-Syntect
+
+First run the following commands on your command line :
+
+```shell
+mkdir ~/.config/xi/dumps
+mkdir ~/.config/xi/syntaxes
+```
```

Comment:
if we need these setup steps they should be part of the make file. As mentioned earlier, though, these paths are not hard coded.